src/index.js

```diff
@@ -6,13 +6,33 @@ import { useMediaPredicate } from 'react-media-hook'
 import apolloClient from 'apolloClient'
 import ReactModal from 'react-modal'
 import PrimaryLayout from 'pages/PrimaryLayout'
+import HFPrimaryLayout from 'pages/holofuel/PrimaryLayout'
 import RegisterUser from 'components/RegisterUser'
 import ScreenWidthContext from 'contexts/screenWidth'
-import './global-styles/normalize.css'
-import './global-styles/colors.css'
-import './index.css'
+
+const HOLOFUEL_APP = true
 
 export function App () {
+  if (HOLOFUEL_APP) {
```

Comment:
using a dumb constant rather than routing or query string or something because it's very simple and does the job :)

Using anything else would potentially increase the  complexity of extracting HoloFuel into its own repo.

Reply:
Sound good. This serves us this sprint as we'll be focused I think entirely on getting the HoloFuel MVP going. We'll revisit later.